output_template/python/lmnet/protobuf_loader.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import tensorflow as tf
+
+
+class ProtobufLoader:
```

Comment:
The class name `ProtobufLoader` is not much what the class is, I think :thinking: 
The class main feature is to execute the graph with input data. For this feature, load tensorflow graph pb file. Please change the name of class, Also the class document.

Reply:
I think `TFGraphLoadPb` is not what @ruimashita san needed...

Do you have any class name @ruimashita ?
In my idea, current implementation is `NNLib`, so how about `PBLib` or `PbLib` ?
Or if we focuse on execution, `PbRunner` or `PbExecutor` ?

Reply:
ummm 🤔 
If I was, I name`TensorflowGraphRunner`, `TensorflowGraphExecutor`, `TensorflowGraphPredictor`

Reply:
Thanks, `TensorflowGraphRunner` is nice.

Reply:
Noted. `TensorflowGraphRunner` It is. Thanks!